packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.css

```diff
@@ -0,0 +1,112 @@
+:host {
+  display: block;
+}
+
+::slotted(*) {
+  display: none;
+}
+
+nav {
+  background: var(--color-white);
+  color: var(--color-link-default);
+  font-size: inherit;
+  padding: 1.6rem 0;
+}
+
+ul {
+  box-sizing: border-box;
+  list-style: square;
+  margin: 0 auto;
+  max-width: 100rem;
+  padding-left: 0.9375rem;
+  padding-right: 0.9375rem;
+  position: relative;
+  width: 100%;
+}
+
+li {
+  display: inline-block;
+  line-height: 1.5;
+  margin: 0.4rem 0;
+  padding: 0.4rem 0 0.4rem 0;
+  vertical-align: middle;
+}
+
+li::after {
+  content: ' › ';
+  display: inline-block;
+  padding: 0 0.56rem;
+}
+
+li:last-child::after {
+  content: '';
+}
+
+a {
+  background-color: transparent;
+  color: var(--color-link-default);
+  display: inline-block;
+  padding: 0.2rem;
+  transition-duration: 0.3s;
+  transition-property: color, background-color, border-color;
+  transition-timing-function: ease-in-out;
+}
+
+a:hover {
+  background: rgba(0, 0, 0, 0.05);
+}
+
+a:focus {
+  outline: 2px solid var(--color-gold-light);
+  outline-offset: 0.2rem;
+}
+
+a[aria-current='page'] {
+  color: var(--color-gray-dark);
+  cursor: default;
+  font-weight: bold;
+  pointer-events: none;
+  text-decoration: none;
+}
+
+.va-nav-breadcrumbs--mobile li {
+  display: none;
+}
+
+.va-nav-breadcrumbs--mobile li:nth-last-child(2) {
+  display: inline-block;
+}
+
+.va-nav-breadcrumbs--mobile li:nth-last-child(2)::before {
+  content: ' \2039 ';
+  display: inline-block;
+  padding: 0 0.24rem;
+}
+
+.va-nav-breadcrumbs--mobile li:nth-last-child(2)::after {
+  content: '';
+  display: none;
+  padding: 0;
+}
```

Comment:
In my previous comment, I linked to an old commit that had this [markdown](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/commit/5ed871aebcf9819ed6743257335b6dee82ee315a#diff-2daf03bb495ad970807e273b85e4b2ccd5bd84d0fa012895def22d21eb4de5d1R40):

> Passing prop `mobileFirstProp` to the `<Breadcrumbs />` component will override `state.mobileShow` and present the back by one mobile breadcrumb link on initial render or subsequent renders. This prop could be used in cases where developers want absolute control over which breadcrumb UI to show, instead of letting the user's viewport width toggle the correct view.

@k80bowman, is this something we want to bring over into the web component? It doesn't appear to be used:

![image](https://user-images.githubusercontent.com/2008881/146233446-8098e060-b0c7-4983-97b5-7ddeb6379554.png)

Our [usage guidance](https://design.va.gov/components/breadcrumbs#usage) seems like it doesn't encourage this mobile styling in desktop, stating:

> On desktop, the breadcrumb path should always start with a link to the home page of the site and end with the current page as the last segment.

Reply:
If it is not being used and our usage guidance doesn't recommend this pattern, then I would say we should not bring it over. We can always add it back if people start requesting it.

Reply:
Removed mobileFirstProp